embeddings/evaluator/submission.py

```diff
@@ -29,8 +29,60 @@ class Submission:
     config: Optional[Dict[str, Any]] = None  # any additional config
 
     @staticmethod
+    def _init_evaluator_kwargs(hparams: Dict[str, Any]) -> Dict[str, Any]:
+        evaluator_kwargs = {}
+        evaluation_mode = hparams.get("evaluation_mode", None)
+        tagging_scheme = hparams.get("tagging_scheme", None)
+
+        if evaluation_mode or tagging_scheme:
+            evaluator_kwargs.update(
+                {
+                    "evaluation_mode": evaluation_mode,
+                    "tagging_scheme": tagging_scheme,
+                }
+            )
+        return evaluator_kwargs
```

Comment:
i would suggest to name it `_get_evaluator_kwargs` because it doesn't initialize anything